jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/projects/jakarta-sample/src/main/java/io/openliberty/sample/jakarta/jsonb/JsonbTransientDiagnostic.java

```diff
@@ -25,6 +25,20 @@
     @JsonbTransient
     private int id;
 
-    @JsonbPropery("fav_lang")
+    @JsonbProperty("fav_lang")
     private String favoriteLanguage;
+    
+    @JsonbProperty("person-id")
+    private int getId() { 
+        // A diagnostic is expected on getId because as a getter, it is annotated with other 
+        // Jsonb annotations while its corresponding field id is annotated with JsonbTransient
+        return id;
+    }
+    
+    @JsonbCreator
+    private int setId(int id) {
```

Comment:
This method should not have a return type

Reply:
Fixed